### PR TITLE
Use latest npm version, and npm ci instead of npm install

### PR DESCRIPTION
**Pull request by sareh**
Use `npm@latest`
Use `npm ci`

We want to use `npm ci`. This is available in `npm@5.6` and later. 
`npm ci` can be significantly faster than a regular `npm install` since it skips some user-facing features. 

Here is a summary of the main differences between `npm ci` and `npm install` https://docs.npmjs.com/cli/ci#description

* [x] Issue linked https://github.com/bbc/simorgh/issues/94
* [x] Up-to-date with latest - `git pull --rebase origin latest`
* [x] Runs locally - `npm run dev` & [http://localhost:7080/](http://localhost:7080/)
* ~[ ] Tests added for new features~ N/A
* [x] Tests pass - `npm test`
* [x] E2E tests pass - `npm run test:e2e` (requires server running see [README](https://github.com/bbc/simorgh#tests) for details)

* [ ] Test engineer approval

Resolves https://github.com/bbc/simorgh/issues/94

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -9,6 +9,13 @@ notifications:
     on_start: change   # default: never
     on_cancel: always # default: always
     on_error: always # default: always
+cache:
+  directories:
+    - ~/.npm
+    - ~/.cache
```
Comment (jamesbhobbs) [inline .travis.yml]: L12-15 seems out of scope, are they necessary? And if not what is the benefit and risk? We don't exactly need to make our build faster atm.

Comment (sareh) [reply .travis.yml]: Agreed. I've removed those lines.

**Review by jamesbhobbs — APPROVED** [review]

**Review by jtart — APPROVED** [review]
